examples/gno.land/r/demo/disperse/disperse.gno

```diff
@@ -0,0 +1,122 @@
+package disperse
+
+import (
+	"std"
+
+	tokens "gno.land/r/demo/grc20factory"
+)
+
+// Check if number of addresses and coins match
+func checkNumAddrValMatch(addresses []std.Address, coins std.Coins) {
+	if len(addresses) != len(coins) {
+		panic(ErrNumAddrValMismatch)
+	}
+}
+
+// Check if caller correctly sent positive amount of coins
+func checkCoinAmount(coins std.Coins) {
+	for _, coin := range coins {
+		if coin.Amount <= 0 {
+			panic(ErrNegativeCoinAmount)
+		}
+	}
+}
+
+// DisperseGnot parses receivers and amounts and sends out gnot
```

Comment:
From what I see, it also returns any accidentally sent coins. Maybe it would be good to document this, or remove this functionality altogether

Reply:
it is present in the [original ethereum smart contract](https://etherscan.io/address/0xD152f549545093347A162Dce210e7293f1452150#code#L1) so we thought it would be best to keep it